fetch/src/main/java/com/indix/gocd/s3fetch/FetchConfig.java

```diff
@@ -39,9 +45,10 @@ public ValidationResult validate() {
             if (env.isAbsent(AWS_SECRET_ACCESS_KEY)) validationResult.addError(envNotFound(AWS_SECRET_ACCESS_KEY));
         }
         if (env.isAbsent(GO_ARTIFACTS_S3_BUCKET)) validationResult.addError(envNotFound(GO_ARTIFACTS_S3_BUCKET));
-        if (StringUtils.isNullOrEmpty(materialLabel))
+        if (StringUtils.isNullOrEmpty(materialLabel)) {
```

Comment:
Just in case, this seems to be AWS StringUtils. Are we using this across the repo? commons-lang3 `StringUtils.isBlank` takes care of all these plus some more.